srv/salt/_modules/public.py

```diff
@@ -51,3 +51,43 @@ def address():
                         ipaddress.ip_network(u'{}'.format(public_network))):
                         return _address
     return ""
+
+
+def ipv6():
+    """
+    Return if any network is IPv6
+    """
+    if 'public_network' not in __pillar__:
+        return ""
+    return _ipnetwork(6)
+
+
+def ipv4():
+    """
+    Return if any network is IPv4
+    """
+    if 'public_network' not in __pillar__:
```

Comment:
nit picking but maybe push this down to `_ipnetwork` so we only need it once?

Reply:
Both ipv6 and ipv4 return strings back to the Salt user.  The _ipnetwork returns bool.

Reply:
But don't `ipv4` and `ipv6` now return either an empty string or a bool? Seems like the same return could be in `_ipnetwork`, reducing code duplication without a change in the public interface.